pkg/apis/servicecatalog/types.go

```diff
@@ -267,27 +261,18 @@ type BindingSpec struct {
 	// InstanceRef is the reference to the Instance this binding is to.
 	// Immutable.
 	InstanceRef kapi.ObjectReference
-	// AppLabelSelector selects the pods in the Binding's namespace that
-	// should be injected with the results of the binding.  Immutable.
-	AppLabelSelector metav1.LabelSelector
```

Comment:
Refresh my memory... we're removing this because people will create PIPs for this info, right?  If so, have considered leaving this here and using it to create the PIP for the user automatically?  Just to reduce the UX pain of having to create too many resources.